### DISCUSSION TITLE
Screen Banding?

### DISCUSSION THREAD

**Discussion by angelokimhui** [ChefKissInc/NootedRed#278] [General]
Hello, I just tried to Hackintosh again after years (lol). And I used this absolutely insane project of yours to have my 5625U's GPU working. But I noticed that there are screen banding on it. And upon looking at the commit history, the patch for workaround was reverted. Is it because of overhead? It doesn't really bother me so much but yeah, just incase you might've missed. Thanks for your dedication anyway!

**Comment by VisualEhrmanntraut** [comment]
The patch was reverted because it was causing a bunch of people's displays to stop working. A corrected patch will be made at a later date.

**Comment by TOTY444** [comment]
Hello everyone, I wanted to know if anyone has managed to get the HDMI port on their notebook to work. If any of you could tell me how to do this, I would be very grateful. Thank you.
This is my 
Specification:
Processor: Intel Core i7-9750H
iGPU: Intel UHD Graphics 630
eGPU: NVIDIA GeForce GTX 1050 (disabled)
RAM: 16 GB DDR4 Bus 2666
Storage: WD Black SN750 NVMe 512 GB and Samsung SSD 860 EVO 500 GB
Wifi: Broadcomm BCM94352Z
Bluetooth: 20702A3
Audio: Realtek ALC257
Ethernet: Realtek RTL8168 Gigabit Ethernet
Touchpad: Synaptics SYNA 2B52
Boot Mode: UEFI GPT
Screen Size: 15.6"
Display Resolution: Full HD 1920 x 1080

**Reply by VisualEhrmanntraut** [reply]
Hi. Your laptop has a dGPU. This means the port is wired to it rather than the iGPU. The only way to change that is if the laptop has a MUX switch, otherwise the dGPU must have a working driver.

**Comment by VisualEhrmanntraut** [comment]
@angelokimhui still relevant now that HWDisplay has been decompiled and reimplemented proper?
